renovate.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": [
+    "config:base",
+    ":dependencyDashboard",
+    "group:linters"
+  ],
+  "git-submodules": {
+    "enabled": true
+  },
+  "lockFileMaintenance": {
+    "enabled": true,
+    "schedule": ["at any time"],
+    "stabilityDays": 0,
+    "dependencyDashboardApproval": true
+  },
```

Comment:
This regenerates the lockfile on request.

Reply:
Huh... that seems useful.  I'm sure that after enough merges the lock file could end up in an unoptimized state.